Jenkinsfile

```diff
@@ -105,6 +106,16 @@ node {
 
             milestone 4
 
+            stage('Health check on STAGE') {
+                        withEnv(["endpoint=${healthcheckEndpoint}"]) {
+                          def httpStatus=sh(returnStdout: true, script: 'sleep 15; curl -m 300 -s -o /dev/null -w "%{http_code}" 10.160.0.137:8088${endpoint}').trim()
```

Comment:
It looks like the load balanced URL is available so could be used here, using an environment variable if the URL is a detail to be kept private.